flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CepOperatorTestUtilities.java

```diff
@@ -66,6 +68,9 @@ public Integer getKey(Event value) throws Exception {
 		boolean isProcessingTime,
 		NFACompiler.NFAFactory<Event> nfaFactory,
 		EventComparator<Event> comparator) {
+
+		OutputTag<Event> lateDataTag = new OutputTag<Event>("late-data", TypeInformation.of(Event.class));
```

Comment:
Could you pass it as parameter? And by default set a null one? I would prefer that we do not depend on the fact that the name in here and in the test are the same. Would prefer using the same object.